src/Common/src/Interop/OSX/Interop.RunLoop.cs

```diff
@@ -26,7 +27,11 @@ internal static partial class RunLoop
         /// <summary>
         /// Starts the current thread's RunLoop. If the RunLoop is already running, creates a new, nested, RunLoop in the same stack.
         /// </summary>
+#if MONO
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+#else
         [DllImport(Interop.Libraries.CoreFoundationLibrary)]
+#endif
```

Comment:
Do have any sense at this point of how prevalent such ifdefs will need to be?  At some point we may want to revisit this, but for now seems reasonable.  Thanks.